features/support/env.rb

```diff
@@ -36,6 +39,10 @@
   skip_this_scenario("Skipping scenario") if Maze.config.os_version == 10
 end
 
+Before('@skip_android_11') do |scenario|
+  skip_this_scenario("Skipping scenario") if Maze.config.os_version == 11
+end
+
```

Comment:
If this is still needed, please show me where (I think it's just a merge conflict thing and should be removed).